src/platform/unix.rs

```diff
@@ -76,6 +99,7 @@ impl<T> EventLoopWindowTargetExtUnix for EventLoopWindowTarget<T> {
 
 /// Additional methods on `EventLoop` that are specific to Unix.
 pub trait EventLoopExtUnix {
+    #[cfg(feature = "x11")]
```

Comment:
This may be a matter of personal preference, but I feel that `#[cfg]` attributes such as this one should go below the doc comment to improve readability.